.github/workflows/release.yml

```
@@ -27,16 +28,20 @@ jobs:
           echo "::set-output name=version::$version"
 
       - name: Check Tag Exists
+        id: check_tag
         shell: bash
         run: |
           if [ $(git tag -l "v${{ steps.determine_version.outputs.version }}") ]; then
-              echo "Error: Existing git tag found. Exiting..."
-              exit 1
+              echo "Existing git tag found. Exiting..."
```

```suggestion
              echo "Existing git tag found. Exiting..."
              echo "do_release=false" >> $GITHUB_OUTPUT
```

Shouldn't matter, really, but the linter I'm PRing in cactbot complained, so...